README.md

```diff
@@ -1,7 +1,34 @@
 # EncompassRest
 Encompass API Client Library for .NET Framework 4.5+ and .NET Standard 1.1+.
 
-More documentation forthcoming.
+## Why does this exist?
+You may wonder why this library exists when Ellie Mae has provided their own [Encompass API .NET Language Bindings](https://github.com/EllieMae/developerconnect-dotnet-bindings).
+
+First, the Encompass API .NET Language Bindings were released long after the API's became available so API users needed a common .NET library for consuming the Encompass API's, hence this library was born.
+
+Secondly, the swagger generated Encompass API .NET Language Bindings are far inferior to the custom crafted and well thought-out EncompassRest library.
```

Comment:
Secondly, the swagger generated Encompass API .NET Language Bindings are less robust than the custom crafted, well thought-out, and thoroughly tested  EncompassRest library.